seaborn/relational.py

```diff
@@ -364,7 +364,10 @@ def parse_hue(self, data, palette, order, norm):
             cmap = None
             limits = None
             levels, palette = self.categorical_to_palette(
-                data, order, palette
+                # List comprehension here is required to
+                # overcome differences in the way pandas
+                # externalizes numpy datetime64
+                [v for v in data], order, palette
```

Comment:
It probably doesn't matter, but I feel like `list(data)` would be more explicit about what's happening. A no-op list comp just looks weird to me.

Reply:
Thanks, I've changed this.